loader/loader.go

```diff
@@ -46,30 +50,13 @@ var (
 		`your script and modules so that they're accessible by k6 from ` +
 		`inside of the container, see ` +
 		`https://k6.io/docs/using-k6/modules#using-local-modules-with-docker.`
-	nothingWorkedLoadedMsg = fileSchemeCouldntBeLoadedMsg +
-		` Additionally it was tried to be loaded as remote module by prepending "https://" to it, ` +
-		`which also didn't work. Remote resolution error: "%s"`
-	errNoLoaderMatched = errors.New("no loader matched")
 )
 
-// noSchemeRemoteModuleResolutionError is returned when a url with no scheme was tried to be
-// resolved and errored out
-type noSchemeRemoteModuleResolutionError struct {
-	err             error // original error
-	moduleSpecifier string
-}
+type unresolvableURLError string
 
-func (n noSchemeRemoteModuleResolutionError) Error() string {
-	return fmt.Sprintf(
-		`Module specifier "%s" was tried to be loaded as remote module by prepending "https://" to it, `+
-			`which didn't work. If you are trying to import a nodejs module, this is not supported `+
-			`as k6 is _not_ nodejs based. Please read https://k6.io/docs/using-k6/modules for more information. `+
-			`Remote resolution error: "%s"`, n.moduleSpecifier, n.err)
-}
-
-// Unwrap returns the wrapped error.
-func (n noSchemeRemoteModuleResolutionError) Unwrap() error {
-	return n.err
+func (u unresolvableURLError) Error() string {
+	// TODO potentially add more things about what k6 supports if users report being confused.
```

Comment:
I really don't like our messages being super huge. Which is why I prefer to err on the side of them being short then giving a lot of information that potentially won't be useful. 

In this case explaining the modules can be absolute file paths, with or with `file://`, `https://` urls, or relative paths (and what they will be relative to) + the `k6/*` stuff is way too much text. That arguably won't even help you enough.

I am somewhat okay with linking to the docs https://grafana.com/docs/k6/latest/using-k6/modules/, but even that seems a bit longish. 

Especially as my expectation is that the most common case will be `buffer` or something similar from nodejs or npm.

Reply:
cc @oleiade ^

Reply:
Are we discussing this message?

```
The moduleSpecifier \"example.com/html\" couldn't be recognised as something k6 supports
```

How about something like :thinking: 

```
The module specifier \"example.com/html\" is unsupported. Supported module modifiers are `file://`, `https://`
```

Reply:
Or even less, gaining from the context:

```
The module specifier \"example.com/html\" is unsupported. Supported are file:// and https://.
```

Reply:
> Are we discussing this message?

Isn't this what you wanted? Or did I misunderstood?

Reply:
I was trying to address this:

>I really don't like our messages being super huge. Which is why I prefer to err on the side of them being short then giving a lot of information that potentially won't be useful.

I mean, the message that we return right now (got it from tests):

```
The moduleSpecifier \"example.com/html\" couldn't be recognised as something k6 supports
```

And suggesting that we consider modifying it to something that contains examples:

```
The module specifier \"example.com/html\" is unsupported. Supported are file:// and https://.
```

Anyway, it was a non-blocking suggestion, so feel free to ignore it and merge as it is :+1:

Reply:
I've merged it for now - I would prefer to get some user feedback on this before doing something.

If you (or anyone else) disagree - please open an issue. 

We can change the message fairly easy - even though it will need to be checked around in tests